index.js

```diff
@@ -1,47 +1,72 @@
-'use strict';
-
-var chalk = require('chalk');
+'use strict';
+
+var chalk = require('chalk');
 var logUtil = require('./logUtil');
 var linesUtil = require('./linesUtil');
 
-var m = {
+// configuration variable
+var config;
+
+try {

```

Comment:
can make it a function....

```
var config = getConfig();
```

Where getConfig is responsible for looking in the two paths.

Keep in mind, the original intent is they will not need to define ALL the colors, simply the ones they want to override.
With this setup, you either get the override, or get the default.

If you go with the one or the other approach, with no override, you will need to add some defensive programming to ensure all the expected fields are provided.